src/ui/ConsoleView.js

```diff
@@ -1,4 +1,5 @@
 import JsonEditor from './json/JsonEditor';
```

Comment:
// @flow strict

Reply:
This will be resolved after rebasing onto d740c75a0b750295dada04c97fdd1121aff5deb7.